lib/knapsack_pro/runners/rspec_runner.rb

```diff
@@ -32,6 +32,7 @@ def self.run(args)
             # instead we pass test files and test example paths to t.rspec_opts
             t.pattern = []
             t.rspec_opts = "#{args} --default-path #{runner.test_dir} #{runner.stringify_test_file_paths}"
+            t.verbose = false
```

Comment:
I'm wondering if this should be an env var so that the user could decide to suppress the output.

If we make it `verbose=false` as a default, then this suggestion here:

```
You can also run the same subset of tests without Knapsack Pro: in the logs, find the command that Knapsack Pro used to invoke the test runner:

rspec  --default-path spec "spec/models/user_spec.rb" "spec/models/comment_spec.rb"
```

https://docs.knapsackpro.com/ruby/troubleshooting/#regular-mode

It won't be possible anymore to copy&paste RSpec command to run tests locally.

Reply:
I modified the code. It will be the most user-friendly to respect the log level. If someone uses >= warn log level, then we suppress the RSpec output with a list of test files to execute.

Reply:
I like the conclusion you came to